src/litserve/loops/continuous_batching_loop.py

```diff
@@ -120,8 +120,6 @@ async def prefill(
         lit_api: LitAPI,
         lit_spec: Optional[LitSpec],
         request_queue: Queue,
-        max_batch_size: Optional[int] = None,
-        batch_timeout: Optional[float] = None,
```

Comment:
this is public class :)

Reply:
yes, but not documented anywhere as of now.

Reply:
even not properly documented, it is public API :)

Reply:
fair enough! added it back.